[x86-64-assembly] Update basics exercise (#1844)
exercism/v3: languages/x86-64-assembly/exercises/concept/basics/.docs/hints.md

## General

- You need to declare the function as [`global`][global].
- The value in the `rax` register specifies the value returned by the function.

## 1. Define the expected oven time in minutes

> - You can use the [`mov`][mov] instruction to store a value in a register.

Reviewer: The hints in these file are really helpful, but they do spell out the solution to the student. As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd):

> The hints should not spell out the solution, but instead point to a resource describing the solution

Our reasoning for this is that we want student to know which resources to use when working with the language (and how to read those resources). This will greatly help the student later on in other exercises and we've found that when we do not spell out the solution, the learning experience is better.

So maybe you could do something like:

```suggestion
- There is an [instruction][mov] to store a value in a register.
```

For inspiration, consider how the C# track's `strings` exercise's [hints.md file](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/hints.md) goed about doing this.

This comment also applies to other hints in this file.
